images/win/scripts/Installers/Install-Vsix.ps1

```diff
@@ -0,0 +1,27 @@
+###################################################################################
+##  File:  Install-Vsix.ps1
+##  Desc:  Install the Visual Studio Extensions from toolset.json
+###################################################################################
+
+Import-Module -Name ImageHelpers -Force
+$ErrorActionPreference = "Stop"
+
+$toolset = Get-ToolsetContent
+$requiredVsixs = $toolset.visualStudio.vsix
+if(-not $requiredVsixs) {
+    Write-Host "No requiered VSIX"
+    exit 1
+}
+
+if (Test-IsWin19) {

```

Comment:
I think we can read this value from toolset: https://github.com/actions/virtual-environments/blob/main/images/win/toolsets/toolset-2019.json#L167

Reply:
has been changed to `$VsVersion = $toolset.visualStudio.Version`